File: _bench/fib/fib.ts
#!/usr/bin/env -S bun run

function fib(n: number) {
  if (n < 2) {
    return 1;
  } else {
    return fib(n - 1) + fib(n - 2);
  }
}
const n = 40;
console.log(fib(n));

File: _bench/fib/fib.c
#include <stdio.h>

int fib(int n) {
  if (n < 2) {
    return 1;
  } else {
    return fib(n-1) + fib(n-2);
  }
}

int main() {
  int n = 40;
  printf("%d\n", fib(n));
  return 0;
}
